### PR TITLE
NIAD-1881: Receive EHR extract response

### PR DESCRIPTION
**Pull request by mrogowska**

### CODE REVIEW

File: common/src/main/java/uk/nhs/adaptors/common/util/DateUtils.java [hunk 1]
```diff
@@ -10,4 +12,7 @@
     public OffsetDateTime getCurrentOffsetDateTime() {
         return OffsetDateTime.now(ZoneOffset.UTC);
     }
+    public Instant getCurrentInstant() {
+        return Instant.now().truncatedTo(ChronoUnit.MILLIS);
```
Comment (dawidskainos) [inline common/src/main/java/uk/nhs/adaptors/common/util/DateUtils.java:16]: I think we should use `atZone(UTC)`

File: common/src/main/java/uk/nhs/adaptors/common/testutil/FileUtils.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package uk.nhs.adaptors.common.testutil;
```
Comment (bchotkowski) [inline common/src/main/java/uk/nhs/adaptors/common/testutil/FileUtils.java:1]: Should we change the name to `FileUtil` like "singular util" to match the naming convention?

File: gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/model/EhrExtractRequest.java [hunk 3]
```diff
@@ -1,13 +0,0 @@
-package uk.nhs.adaptors.pss.translator.model;
```
Comment (bchotkowski) [inline gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/model/EhrExtractRequest.java:1]: Thanks for deleting it! :)

File: gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/EhrExtractHandlingIT.java [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+@SpringBootTest(webEnvironment = RANDOM_PORT)
+@ExtendWith({SpringExtension.class})
+@DirtiesContext
+@AutoConfigureMockMvc
+public class EhrExtractHandlingIT {
+    private static final int NHS_NUMBER_MIN_MAX_LENGTH = 10;
+    private static final String EBXML_PART_PATH = "xml/ebxml_part.xml";
+    private static final String NHS_NUMBER_PLACEHOLDER = "{{nhsNumber}}";
+    private static final List<String> IGNORED_JSON_PATHS = List.of(
+        "id",
```
Comment (dawidskainos) [inline gp2gp-translator/src/integrationTest/java/uk/nhs/adaptors/pss/translator/EhrExtractHandlingIT.java:46]: I think this exclusion list is too big - most of IDs are taken from HL7, only few are random

**Review by dawidskainos — CHANGES_REQUESTED** [review]

**Review by bchotkowski — APPROVED** [review]

**Review by dawidskainos — APPROVED** [review]

**Pull request merged**